playwright/src/test/java/com/microsoft/playwright/TestPdf.java

```diff
@@ -50,16 +50,8 @@ void shouldSupportFractionalScaleValue(@TempDir Path tempDir) throws IOException
 
   @Test
   @DisabledIf(value="com.microsoft.playwright.TestBase#isChromium", disabledReason="skip")
-  void shouldOnlyHavePdfInChromium() {
-    try {
-      page.pdf();
-      if (isChromium()) {
-        return;
-      }
-      fail("did not throw");
-    } catch (PlaywrightException e) {
-      assertFalse(e.getMessage().contains("did not throw"));
-    }
+  void shouldThrowInNonChromium() {
+    PlaywrightException e = assertThrows(PlaywrightException.class, () -> page.pdf());
+    assertFalse(e.getMessage().contains("did not throw"));
```

Comment:
this is not necessary any more, let's replace it with a check for the actual message.

Reply:
Done